Workiva/wf-uicomponents
# HY-1795 fix blurry SVG content on zoom

# Problem
Zooming in using Chrome 39+ leaves SVG content blurry. 

# Solution
After a transition, briefly apply willChange: contents and schedule immediate the removal of that property. This causes the SVG content to be redrawn and recomposited.

# Testing
- View a document in Chrome desktop and zoom in. Verify the content becomes blurry.
- bower link this PR branch in, grunt and reload (check you have the updated code)
- Verify zooming in results in crisp SVG content.
- Verify scrolling performance is still great
